.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!-- Thanks for submitting a pull request! We appreciate you spending the time to work on these changes. Please provide enough information so that others can review your pull request. The three fields below are mandatory. -->
+
+## Summary
+
+<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? -->
+
```

Comment:
In summary, I think it would be good to add subtitles like [this link](https://github.com/kubernetes/kubernetes/pull/111583)

So, how about changing like this?
```suggestion
#### What type of PR is this?
<!--
Add one of the following kinds:
/kind bug
/kind test
/kind style
/kind chore
/kind feature
/kind refactor
/kind documentation
-->

#### What this PR does / why we need it:

#### Which issue(s) this PR fixes:
<!--
*Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*
-->
Fixes #
```